vars/imagePushToRegistry.groovy

```diff
@@ -0,0 +1,9 @@
+import groovy.transform.Field
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/imagePushToRegistry.yaml'
+
+void call(Map parameters = [:]) {
+    List credentials = [[type: 'file', id: 'dockerConfigJsonCredentialsId', env: ['PIPER_dockerConfigJSON']]]
```

Comment:
Support other Jenkins credentials (???)

Reply:
i think we should encourage vault otherwise part of the credentials are in Jenkins and part in vault, it will be confusing. for a starter lets avoid it